openedx/frontend-app-authoring
# feat: improve collection sidebar

## Description

This PR implements the `Details` tab in the Collection Sidebar and change the behaviour while clicking in a Collection Card in the Library Home.

![image](https://github.com/user-attachments/assets/648dc4f0-2068-4c13-a276-5a98cb154384)

## More Information

Part of:
 - https://github.com/openedx/frontend-app-authoring/issues/1089 

## Testing Instruction
- Open the Library Authoring Home for a library
- Add a new collection
- Click on the newly created Collection Card: this should open the sidebar
- Click on the `Details` tab
- Using the sidebar, edit the collection's name. You should see the new name in the sidebar and the collection list. The `Last Modified` date should also be updated.
Using the sidebar, edit the Collection description. You should see the new description in the sidebar and the collection list. The `Last Modified` date should also be updated.
- Click the  `Open` button, and you should be redirected to the Collection Page
- You can also navigate to the Collection Page using the menu from the Collection Card

___
Private ref: [FAL-3791](https://tasks.opencraft.com/browse/FAL-3791)
